mmv1/main.go

```diff
@@ -33,6 +33,9 @@ var yamlMode = flag.Bool("yaml", false, "copy text over from ruby yaml to go yam
 // Example usage: --template
 var templateMode = flag.Bool("template", false, "copy templates over from .erb to go .tmpl")
 
+// Example usage: --handwritten
+var handwrittenMode = flag.Bool("handwritten", false, "copy handwritten files over from .erb to go .tmpl")
```

Comment:
We should add this to the GNUMakefile command `make refresh-go`

Reply:
Sure. I will add it there after the first round is done.